### PR TITLE
Upade service page

### PR DESCRIPTION
**Pull request by alfredodev**

### CODE REVIEW

**Comment by MiyamotoAkira** [comment]
Aigh to the mix of spanish and english. Back when I was living in Spain I disliked it. That hasn't changed.

File: src/_i18n/es.yml [hunk 1]
```diff
@@ -141,9 +141,14 @@ publications-post:
 services:
   title: Software de calidad entregado efectivamente
   subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
+  we-acd: We advise, we coach, we deliver
+  software-craftsmanship-pioneers: Somos pioneros en Software Craftsmanship. Somos conocidos por nuestra experiencia en prácticas XP, Continuous Delivery, Software Architecture, soluciones Cloud e Infraestructura. Aprovechamos nuestra experiencia para ofrecer un servicio a medida de nuestros clientes.
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml]: mixing cases here (soluciones Cloud)

File: src/_i18n/es.yml [hunk 2]
```diff
@@ -141,9 +141,14 @@ publications-post:
...
+  we-advise-subtitle: Te asesoramos en la entrega de Software de calidad de forma efectiva.
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml]: Software should be lowercase

File: src/_i18n/es.yml [hunk 3]
```diff
@@ -141,9 +141,14 @@ publications-post:
 services:
   title: Software de calidad entregado efectivamente
   subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
+  we-acd: We advise, we coach, we deliver
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml:144]: Asesoramos, ensenamos, desarrollamos.   ????

File: src/_i18n/es.yml [hunk 4]
```diff
@@ -141,9 +141,14 @@ publications-post:
...
+  software-craftsmanship-pioneers: Somos pioneros en Software Craftsmanship. Somos conocidos por nuestra experiencia en prácticas XP, Continuous Delivery, Software Architecture, soluciones Cloud e Infraestructura. Aprovechamos nuestra experiencia para ofrecer un servicio a medida de nuestros clientes.
+  we-advise-subtitle: Te asesoramos en la entrega de Software de calidad de forma efectiva.
+  we-coach-subtitle: Te entrenamos en habilidades de diseño de Software y delivery.
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml]: anyway, software -> programas? applicaciones?

Comment (MiyamotoAkira) [inline src/_i18n/es.yml]: delivery -> desarrollo

File: src/_i18n/es.yml [hunk 5]
```diff
@@ -141,9 +141,14 @@ publications-post:
...
+  we-deliver-subtitle: Entregamos soluciones de Software de calidad.
   learn-more: Más información
-  expert-advice:
-    title: Consejo experto
+  we-advise:
+    title: We advise
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml:152]: As per title above

File: src/_i18n/es.yml [hunk 6]
```diff
@@ -171,12 +176,15 @@ services:
...
         Ofrecemos formación y coaching en prácticas XP
         (ej. TDD, Pair programming, Continuous Integration), Software Design, Clean Code y Arquitectura de Software.
-    clients-title: Clientes que han recibido consejo experto
-  training:
-    title: ¿Estas buscando formación?
-    body: Ayudamos a equipos en cualquier fase de un proyecto donde se puedan beneficiar de formación focalizada.
-  effective-delivery:
-    title: Entrega efectiva
+  we-coach:
+    title: We coach
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml:181]: As per title above

File: src/_i18n/es.yml [hunk 7]
```diff
@@ -195,14 +203,11 @@ services:
...
+    title: Nuestros clientes
   our-expertise:
     title: Nuestra experiencia
-    software-crasftsmanship: Software Craftsmanship
-    agile-and-lean-processes: Agile and Lean Processes
-    continuous-delivery: Continuous Delivery
-    extreme-programming: Extreme Programming
-    devops-culture: DevOps Culture
-    technology-focused: Technology Focused
+    subtitle: Nuestra oferta y experiencia se basan en valores Software Craftmanship, en prácticas XP, en cultura Devops y en procesos Agile.
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml]: Devops -> DevOps ??

File: src/_i18n/es.yml (hunk 4 above)
Comment (MiyamotoAkira) [reply src/_i18n/es.yml]: entrenar in Spanish is only for physical teaching.

File: src/_i18n/es.yml (hunk 7 above)
Comment (MiyamotoAkira) [reply src/_i18n/es.yml]: en valores de Software Craftsmanship

File: src/_i18n/es.yml [hunk 8]
```diff
@@ -141,9 +141,14 @@ publications-post:
 services:
   title: Software de calidad entregado efectivamente
   subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml]: Pericia is not that used in Spanish (other than people that have played D&D in the Zinco print of the 2nd edition)

Comment (MiyamotoAkira) [reply src/_i18n/es.yml]: Habilidad?

**Comment by MiyamotoAkira** [comment]
Other than the comments about the spanish translation, the page looks fine.

**Comment by MiyamotoAkira** [comment]
Banner at the top is left aligned. Looks very strange with every other title being centered

File: src/_i18n/es.yml (hunk 4 above)
Comment (alfredodev) [reply src/_i18n/es.yml]: I'm leaving Software as I don't like the alternatives.
i'm changing `Te entrenamos` for `Te formamos`

File: src/_i18n/es.yml (hunk 3 above)
Comment (alfredodev) [reply src/_i18n/es.yml:144]: I'm leaving it for now until we find a better alternative, also I'm not sure if it's our new motto and we should translate it.

File: src/_i18n/es.yml (hunk 4 above)
Comment (alfredodev) [reply src/_i18n/es.yml]: In this case I think is more entrega than desarrollo... but I dislike both, I think is better to leave it in english

File: src/_i18n/es.yml [hunk 9]
```diff
@@ -140,12 +140,12 @@ publications-post:
...
 services:
   title: Software de calidad entregado efectivamente
-  subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
+  subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y habilidad.
   we-acd: We advise, we coach, we deliver
-  software-craftsmanship-pioneers: Somos pioneros en Software Craftsmanship. Somos conocidos por nuestra experiencia en prácticas XP, Continuous Delivery, Software Architecture, soluciones Cloud e Infraestructura. Aprovechamos nuestra experiencia para ofrecer un servicio a medida de nuestros clientes.
-  we-advise-subtitle: Te asesoramos en la entrega de Software de calidad de forma efectiva.
-  we-coach-subtitle: Te entrenamos en habilidades de diseño de Software y delivery.
-  we-deliver-subtitle: Entregamos soluciones de Software de calidad.
+  software-craftsmanship-pioneers: Somos pioneros en software Craftsmanship. Somos conocidos por nuestra experiencia en prácticas XP, Continuous Delivery, Software Architecture, Soluciones Cloud e Infraestructura. Aprovechamos nuestra experiencia para ofrecer un servicio a medida de nuestros clientes.
```
Comment (MiyamotoAkira) [inline src/_i18n/es.yml]: if software is lowercase, craftsmanship should be too. if the next line is all capitalized, also this should be.

Comment (MiyamotoAkira) [reply src/_i18n/es.yml]: Probably I will leave it capitalized.

**Pull request merged**